### PR TITLE
Yii module improvements

### PR DESCRIPTION
**Pull request by DavertMik**
* added `init` part to make module work for acceptance and unit tests
* server vars are now set by module from a config
* `entryScript` and `entryConfig` values added

TODO:

- [x] add methods to work with routes
- [x] add `grabService` method
- [x] add fixtures methods
- [x] add email assertion method
- [x] add quick login method

### CODE REVIEW

File: RoboFile.php [hunk 1]
```diff
@@ -815,7 +815,7 @@ public function publishBase($branch = null, $tag = null)
 
         $this->taskComposerUpdate()->run();
         $this->taskGitStack()
-            ->add('composer*')
```
Comment (Naktibalda) [inline RoboFile.php]: This is not a yii improvement.

**Comment by Naktibalda** [comment]
@DavertMik Could you use more meaningful commit messages than "Improved" and "improvements"?

**Comment by DavertMik** [comment]
> Could you use more meaningful commit messages than "Improved" and "improvements"?

I wrote the wider description in Pull Request. Writing this inside a terminal was not that convenient, actually

File: RoboFile.php (hunk 1 above)
Comment (DavertMik) [inline RoboFile.php]: yes, I mentioned it. I will try to cherry pick that and move to another PR

**Comment by DavertMik** [comment]
Once I finish all issues I can rebase and add this description as commit message

File: src/Codeception/Module/Yii2.php [hunk 2]
```diff
@@ -89,22 +150,137 @@ public function _after(\Codeception\TestInterface $test)
...
+    {
+        return call_user_func_array('array_merge',
+            array_map(function($fixturesStore) {
+                return $fixturesStore->getFixtures();
+            })
+        );
+    }
+
+    /**
+     * Gets a fixtures by name.
```
Comment (samdark) [inline src/Codeception/Module/Yii2.php]: fixtures → fixture

File: src/Codeception/Module/Yii2.php [hunk 3]
```diff
@@ -442,6 +442,62 @@ public function grabComponent($component)
...
+    /**
+     * Checks that no email was sent
+     */
+    public function dontSeeEmailIsSent()
+    {
+        $this->seeEmailIsSent(0);
+    }
+
+    /**
+     * Returns array fo all sent email messages.
```
Comment (samdark) [inline src/Codeception/Module/Yii2.php]: fo → of

File: src/Codeception/Lib/Connector/Yii2/Logger.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Codeception\Lib\Connector\Yii2;
+
+use Codeception\Util\Debug;
+
+class Logger extends \yii\log\Logger
+{
+    public function init()
+    {
```
Comment (samdark) [inline src/Codeception/Lib/Connector/Yii2/Logger.php]: I'd add comment that empty method is intentional i.e.:

```
// prevent shutdown function registration
```

File: src/Codeception/Lib/Connector/Yii2/Logger.php [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    public function log($message, $level, $category = 'application')
+    {
+        if (!in_array($level, [
+            \yii\log\Logger::LEVEL_INFO,
+            \yii\log\Logger::LEVEL_WARNING,
+            \yii\log\Logger::LEVEL_ERROR,
+        ])) {
+            return;
+        }
+        if (strpos($category, 'yii\db\Command')===0) {
```
Comment (samdark) [inline src/Codeception/Lib/Connector/Yii2/Logger.php]: Could be done via config (`except`) property:

http://www.yiiframework.com/doc-2.0/guide-runtime-logging.html#message-filtering

**Pull request merged**